engine/pong.cpp

```diff
@@ -69,6 +74,7 @@ void Pong::Init()
     GetInstance().mGameObjects.push_back(std::move(scoreController));
 
     Pong::GetInstance().GetTimer().Init();
+    ASSERT(Pong::GetInstance().GetAudioMixer().Init());
```

Comment:
I'd possibly be inclined to put the asserts down in this function and die immediately at the source of the problem, but this is OK as well!

Reply:
Yeah, that does bring more immediate attention to where it failed